tests/common/test_cgroupapi.py

```diff
@@ -18,16 +18,16 @@
 from __future__ import print_function
 
 import subprocess
-
-from azurelinuxagent.common.cgroupapi import CGroupsApi, FileSystemCgroupsApi, SystemdCgroupsApi, VM_AGENT_CGROUP_NAME
-from azurelinuxagent.common.exception import ExtensionError, ExtensionErrorCodes
+from azurelinuxagent.common.cgroupapi import CGroupsApi, FileSystemCgroupsApi, SystemdCgroupsApi, CGROUPS_FILE_SYSTEM_ROOT, VM_AGENT_CGROUP_NAME
+from azurelinuxagent.common.exception import CGroupsException, ExtensionError, ExtensionErrorCodes
 from azurelinuxagent.common.future import ustr
 from azurelinuxagent.common.utils import shellutil
 from nose.plugins.attrib import attr
+from tests.utils.cgroups_tools import CGroupsTools
 from tests.tools import *
```

Comment:
Refactor `import *`?

Reply:
Looked into it when I started modifying this test. There is some funny stuff there that is going to require more time than I have the current release.

Let's see who in the team gets there first.